client/login/controller.js

```diff
@@ -88,6 +89,14 @@ export function magicLogin( context, next ) {
 	next();
 }
 
+export function qrCodeLogin( context, next ) {
+	const { path } = context;
+
+	context.primary = <QrCodeLogin path={ path } />;
```

Comment:
OTOH, we seem to be using `locale` in the component, did we mean to pass `locale={ context.params.locale }`?

Reply:
Good call.